dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/CDIContainerImpl.java

```diff
@@ -73,7 +76,8 @@
 
     private static final String EXTENSION_API_CLASSES_SEPARATOR = ";";
 
-    private Set<ExtensionArchive> runtimeExtensionSet = null;
+    private final Map<Long, ExtensionArchive> runtimeExtensionMap = new HashMap<>();
```

Comment:
Add javadoc to explain the meaning of the key

Reply:
done